src/misclibs.F

```diff
@@ -4554,50 +4339,30 @@ subroutine tmf(nstat,rstat,ruh,rvh,rho,wa)
 
       integer i,j,k
       double precision :: tmfu,tmfd,mf,var
-      double precision, dimension(nk) :: foo1,foo2
       double precision :: foo1t,foo2t
 
-     !$acc data create(foo1,foo2)
 
-     !$omp parallel do default(shared) private(i,j,k,mf)
-#ifdef _B4B10R
-     !$acc update host(wa,ruh,rvh)
-#else
-     !$acc parallel default(present)  private(i,j,k,mf) reduction(+:foo1t,foo2t) &
-     !$acc reduction(+:tmfu,tmfd)
-     !$acc loop gang 
-#endif
+      tmfu=0.0d0
+      tmfd=0.0d0
+      !$omp parallel do default(shared) private(i,j,k,mf)
+      !$acc parallel default(present)  private(i,j,k,mf) reduction(+:foo1t,foo2t) &
+      !$acc reduction(+:tmfu,tmfd)
+      !$acc loop gang 
```

Comment:
Do we need to add `reduction(+:tmfu,tmfd)` here?

Reply:
Yes we do

Reply:
Unfortunately, the addition of a reduction() clause generates an internal compiler error.  So I reverted the code to the original loop structure.